core/modules/views/includes/views.views.inc

```diff
@@ -143,7 +143,13 @@ function views_views_plugins() {
         'theme' => 'views_view',
         'no ui' => TRUE,
         'no remove' => TRUE,
-        'js' => array('core/misc/form.js', 'core/misc/collapse.js', 'core/misc/textarea.js', 'core/misc/tabledrag.js', 'core/misc/autocomplete.js', 'core/misc/states.js'),
+        'js' => array(
+          'core/misc/form.js',
+          'core/misc/collapse.js',
+          'core/misc/tabledrag.js',
+          'core/misc/autocomplete.js',
+          'core/misc/states.js',
+        ),
```

Comment:
If the order here does not matter, can we please list things alphabetically?

Reply:
I don't actually know if order matters here, so leave it as-is, until you point me to the documentation, whether it matters or not.